gdal/ogr/ogrsf_frmts/gpkg/ogrgeopackagedatasource.cpp

```diff
@@ -352,6 +346,29 @@ OGRSpatialReference* GDALGeoPackageDataset::GetSpatialRef(int iSrsId,
         return oIter->second;
     }
 
+    if( iSrsId == 0 || iSrsId == -1 )
+    {
+        OGRSpatialReference *poSpatialRef = new OGRSpatialReference();
```

Comment:
You'll perhaps need to add
```
poSpatialRef->SetAxisMappingStrategy(OAMS_TRADITIONAL_GIS_ORDER);
```

Reply:
Thanks, added.

Of course, I'll look into the unit tests.